types/react-image-crop/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-image-crop 7.0
+// Type definitions for react-image-crop 8.0
```

Comment:
When upgrading the major version, please consider the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

Reply:
Hi, thanks for reviewing! 

Do you suggest that i create a v7 folder, and move all the old files into that folder? The guide says; 

> If you intend to continue updating the older version of a library's type declarations, you may create a new subfolder

But i am not sure that the v7 is ever going to be updated, so is it necessary to create the subfolder? I am a bit confused about when to do what. What do you think. :-)

Reply:
.. I have added a subfolder for the v7.